src/Kdyby/Translation/Translator.php

```diff
@@ -130,6 +137,7 @@ public function translate($message, $count = NULL, $parameters = array(), $domai
 			return $message;
 
 		} elseif ($message instanceof Nette\Utils\Html) {
+			$this->logger->log($message, static::LOGGER_NAMESPACE);
```

Comment:
Well, it makes sense to inform the developer on the page, but there might be cases where this is intended and it would just spam the log.